src/main/java/frc/robot/vision/cameras/limelight/Limelight.java

```diff
@@ -23,11 +23,11 @@ public class Limelight implements ObjectDetector, IndependentRobotPoseSupplier,
 
 	private final String name;
 	private final String logPath;
-	private final Pose3d robotRelativeCameraPose;
+	private final Pose3d originalRelativeCameraPose;
 	private final ArrayList<DetectedObjectObservation> detectedObjectObservations;
 
 	private LimelightTarget2dValues target2dValues;
-
+	private Pose3d robotRelativeCameraPose;
```

Comment:
```suggestion
	private Pose3d currentRobotRelativeCameraPose;
```
also move above target2dValues